examples/pytorch/image-classification/run_image_classification_no_trainer.py

```diff
@@ -439,7 +439,9 @@ def collate_fn(examples):
         name=args.lr_scheduler_type,
         optimizer=optimizer,
         num_warmup_steps=args.num_warmup_steps * accelerator.num_processes,
-        num_training_steps=args.max_train_steps if overrode_max_train_steps else args.max_train_steps * accelerator.num_processes,
+        num_training_steps=args.max_train_steps
```

Comment:
This change is because I run `make quality`. Are changes like this in this PR intended?

Reply:
This is OK. This is from another PR which was recently merged in. The line is splitting because it's over the limit of 120 characters. 

I'm not sure why it wasn't flagged in the other PR, but it's fine to leave these diffs in.